src/lazy/expanded/macro_evaluator.rs

```diff
@@ -419,20 +420,19 @@ impl<'top, D: Decoder> ValueExpr<'top, D> {
 
 /// Indicates which of the supported macros this represents and stores the state necessary to
 /// continue evaluating that macro.
-#[derive(Copy, Clone, Debug)]
+#[derive(Debug)]
```

Comment:
🪧 `flatten` is the first system macro to be implemented which requires a macro evaluator to persist across calls to `evaluator.next()`. As a result, the `MacroExpansion` family of types can no longer implement `Copy`, as each one might own a bump-allocated evaluator of its own. The `Copy` impl was only being used directly in one speculatively defined function that had no callers. I'll point it out later in the diff.